README.md

```diff
@@ -56,167 +59,125 @@ For updates on the latest meetup, please check us out on [Meetup.com](http://www
-Requires a log in. [Sign up here, under "Non-Singtel Customers"](http://info.singtel.com/personal/internet/broadband-on-the-go/wireless-sg). There is also a second network SSID called Wireless@SGx, which uses WPA-Enterprise + TKIP for authentication.
+[Wireless@SG](https://www.imda.gov.sg/how-we-can-help/wireless-at-sg/wireless-at-sg-for-consumers) is an open network available at almost every shopping mall, library and public space.
 
-And the reboutable ladyironchef [lays down](http://www.ladyironchef.com/2013/08/cafes-free-wifi-singapore/)
-where the free wifi cafes are.
+You can also consider checking out the [cafes with free wi-fi](http://www.ladyironchef.com/2013/08/cafes-free-wifi-singapore/)
 
 ## Goods and Services Tax (GST)
 
-Do take note that Singapore imposes a 7% levy on nearly all forms of goods and services rendered/sold on the island. This price is mostly (if not, always) included in the listed price. However, some establishments may still leave out the tax on listed pricings.
-
-Hence, do still keep a lookout for text at the bottom of menus and product listings that indicate the non-addition of tax and service charge and calculate accordingly.
+Do take note that Singapore imposes a 9% levy on nearly all forms of goods and services. The prices for goods almost always include the GST. The restaurants menus however usually exclude the GST and service charge. It's common practice for restaurants to include a text at the bottom of menus for "++" which indicate additional tax and service charge respectively.
 
 ## Tipping
 
-No real need to tip. Leave a few coins if you'd like (to get rid of your coins).
+It's not expected to leave a tip in Singapore. Although feel free to leave a few coins at the end of your meal if you feel like it.
 
-You don't need to tip at a hawker food stall, and normal restaurants include a
-10% service charge.
-
-[Comment: Oh and talking about coins, Singapore is in a transition from one set
-of coins to another differently sized set. So you'll get two sizes of each
-coin. Yay!]
+Hawker centers are mostly self-service and don't have service charges. Restaurants typically add 10% service charge to your final bill.
 
 ## Tissue Culture
 
-Singaporeans often leave tissues on seats in busy food halls (hawker centres
-etc.) while they walk around and buy food. I know you'll be tempted but best
-not to remove the tissue and sit down.
+Singaporeans often leave tissues on tables in busy food courts and hawker centers before they order their food to "reserve" the table. You might be tempted to remove it, but don't. Just find another table and use your own tissue to reserve it.
 
-[Comment: If you need your own tissues, hawker centres are the place to buy
-them. You'll often find an older gentleman or lady offering one, two or more
-packets for a dollar or two.]
+If you don't have a tissue, most hawker centers have an old lady or gentleman offering for a dollar or two, feel free to buy from them.
```

Comment:
I think the drink stores of all hawker centers / food courts / coffee shop sells a packet of tissue for roughly $0.30 too.